src/main/resources/db/migration/V1.17.0__Create_break_flag_table.sql

```diff
@@ -0,0 +1,7 @@
+use SeichiAssist;
+
+CREATE TABLE break_flags(
+    uuid CHAR(36) NOT NULL PRIMARY KEY,
+    flag_name VARCHAR(20) NOT NULL,
+    can_break BOOL NOT NULL DEFAULT TRUE
```

Comment:
```suggestion
    include BOOL NOT NULL DEFAULT TRUE
```

nit: プレイヤーが直接可能かどうかを決定するわけではないのでcanは変かもしれない